src/components/signoff/SignoffToolBar.tsx

```diff
@@ -110,10 +110,27 @@ export default function SignoffToolBar({
   const { status } = source;
 
   const canRequestReview = canEdit && isEditor(source, sessionState);
+
+  let toReviewEnabled =
+    sessionState.serverInfo?.capabilities?.signer?.to_review_enabled === true;
+  if (toReviewEnabled) {
+    const resourceMatch =
+      sessionState.serverInfo?.capabilities?.signer?.resources?.find(
+        x =>
+          x.source.bucket === source.bid &&
+          x.source.collection === source.cid &&
+          x.destination.bucket === destination.bid &&
+          x.destination.collection === destination.cid
+      );
+    toReviewEnabled =
+      !resourceMatch || resourceMatch.to_review_enabled !== false;
+  }
```

Comment:
There is one case missing: when review is enabled/disabled at the bucket level:
```js
if (!resourceMatch) {
  // No specific config for this collection.
  // Check at bucket level.
  resourceMatch =
    sessionState.serverInfo?.capabilities?.signer?.resources?.find(
      x =>
        x.source.bucket === source.bid &&
        x.source.collection === null &&
        x.destination.bucket === destination.bid &&
        x.destination.collection === null
    );
}
```

But it is not used in practice, so I guess we can leave it like this. Maybe add a code comment.

Example in python:
https://github.com/mozilla-services/remote-settings-lambdas/blob/5c261be0bd927a0ce8af79f045c9eab4ae23a9d2/commands/backport_records.py#L102-L126